src/map/party.h

```diff
@@ -27,8 +27,8 @@
 
 #include <stdarg.h>
 
-#define PARTY_BOOKING_JOBS 6
-#define PARTY_BOOKING_RESULTS 10
+#define MAX_PARTY_BOOKING_JOBS 6
```

Comment:
why you on this defines, can you add also guard header for changed defines?
for example ``#define MAX_PARTY_BOOKING_JOBS 6`` should be changed to
```
#ifndef MAX_PARTY_BOOKING_JOBS
#define MAX_PARTY_BOOKING_JOBS 6
#endif
```

same for ``MAX_PARTY_BOOKING_RESULTS``